### ISSUE TITLE
Use ethers.js

### ISSUE THREAD

**Issue by dOrgJelli** [daostack/client#418]

**Comment by tibetsprague** [comment]
why? for what?

**Comment by jellegerbrandy** [comment]
because react-native and web3 hate each other.

**Comment by dOrgJelli** [comment]
Yes like Jelle said, Web3.js includes several dependencies internally that don't play well with Webpack, Babel, and other build / packaging pipelines.

Also the bundle size of Web3 is very large in comparison to Ethers.js.
